packages/twenty-server/src/engine/api/graphql/workspace-query-runner/jobs/call-webhook.job.ts

```diff
@@ -46,10 +47,11 @@ export class CallWebhookJob {
       const eventInput = {
```

Comment:
This part seems a bit redundant as well, shouldn't success be dependant on the response status / call failure? The rest seems very similar so I think it could be refactored a bit